### PR TITLE
Added lame and ffmpeg.

### PR DESCRIPTION
**Pull request by Rinnegatamante**

### CODE REVIEW

**Comment by SonicMastr** [comment]
Thanks for the contribution. Looks familiar though

File: ffmpeg/VITABUILD [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  ./configure --prefix=$prefix \
+		--enable-cross-compile \
+		--cross-prefix=$VITASDK/bin/arm-vita-eabi- \
+		--disable-shared \
+		--disable-runtime-cpudetect \
+		--disable-armv5te \
+		--disable-programs \
+		--disable-doc \
+		--disable-network \
+		--enable-gpl \
```
Comment (d3m3vilurr) [inline ffmpeg/VITABUILD:24]: an application developers have to get a chance to decide their licence. but `--enable-gpl` option could make the problem to it.

Comment (frangarcj) [reply ffmpeg/VITABUILD]: Agree. From their documentation

Compile FFmpeg without "--enable-gpl" and without "--enable-nonfree".

Comment (Rinnegatamante) [reply ffmpeg/VITABUILD]: I've already pushed a commit that removes it. Without those, ffmpeg will be compiled with LGPL license.

File: lame/VITABUILD [hunk 2]
```diff
@@ -0,0 +1,19 @@
+pkgname=lame
+pkgver=3.100
+pkgrel=1
+url="https://lame.sourceforge.net/"
+source=("https://sourceforge.net/projects/lame/files/lame/$pkgver/lame-$pkgver.tar.gz")
+sha256sums=('ddfe36cab873794038ae2c1210557ad34857a4b6bdc515785d1da9e175b1da1e')
+
+build() {
+  cd $pkgname-$pkgver
+  export CFLAGS="-Wl,-q -O2 -ftree-veectorize -mfpu=neon -fomit-frame-pointer -ffast-math -march=armv7-a -mfloat-abi=hard"
```
Comment (d3m3vilurr) [inline lame/VITABUILD:10]: `-ftree-vectorize` typo

File: ffmpeg/VITABUILD [hunk 3]
```diff
@@ -0,0 +1,54 @@
+pkgname=ffmpeg
+pkgver=3.3.9
+pkgrel=1
+url="http://ffmpeg.org/"
+source=("http://ffmpeg.org/releases/ffmpeg-$pkgver.tar.xz" "ffmpeg.patch")
+sha256sums=('ae34f14fffa65a1a59b256737ca9af7bf4e296b7c4320d42512350126ce06c84' 'c4058a723957547c5450e4c2694eb0b9393d983146079382f311be77f9d87f75')
```
Comment (d3m3vilurr) [inline ffmpeg/VITABUILD:6]: `ffmpeg.patch`'s hashsum is mismatched
`9415e2b829df6c3d52b46113a3e3d6bc823edce079a459f7fca68e7446ffa6e3`

File: ffmpeg/VITABUILD [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
```
Comment (d3m3vilurr) [inline ffmpeg/VITABUILD:7]: this package would be required `depends` packages for building.

Comment (Rinnegatamante) [reply ffmpeg/VITABUILD:7]: Sorry, what do you mean?

Comment (d3m3vilurr) [reply ffmpeg/VITABUILD:8]: imo, this package requires lame. build system couldn't build without depends keyword.

**Pull request merged**